components/RulesList/index.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+import RuleCard from "../RuleCard";
+
+export default function RulesList({ rules }) {
+  console.log(rules);
+  return (
+    <>
+      <StyledUl>
+        {rules.length === 0 && (
```

Comment:
in this case you render an empty list with a `<p>` tag inside. Children of lists should be `<li>`elements. I recommend to check the condition earlier and then render either a list or the message.